### PR TITLE
Customizing retry parameters (fixes #44)

### LINKED ISSUE #44: Customizing retry parameters
I would like to customize request retry parameters to reduce job run time.

```js
  request: {
    retries: 4,
    retryAfter: 60,
  },
```

https://github.com/tgymnich/fork-sync/blob/master/src/main.ts#L10-L13

### PR DESCRIPTION
**Pull request by TheMisir**
Fixes #44

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -22,6 +15,16 @@ async function run() {
   const prMessage = core.getInput('pr_message', { required: false });
   const ignoreFail = core.getInput('ignore_fail', { required: false });
   const autoApprove = core.getInput('auto_approve', { required: false });
+  const retries = parseInt(core.getInput('retries', { required: false })) || 4;
+  const retryAfter = parseInt(core.getInput('retry_after', { required: false })) || 60;
+  
+  const octokit = new MyOctokit({
+    auth: token,
+    request: {
+      retries,
```
Comment (mmorel-35) [inline src/main.ts:24]:
```suggestion
      retries: retries,
```

File: src/main.ts [hunk 2]
```diff
@@ -22,6 +15,16 @@ async function run() {
...
+      retryAfter,
```
Comment (mmorel-35) [inline src/main.ts:25]:
```suggestion
      retryAfter:  retryAfter
```

File: src/main.ts (hunk 1 above)
Comment (mmorel-35) [reply src/main.ts:24]: Have you tried this?

File: src/main.ts (hunk 2 above)
Comment (mmorel-35) [reply src/main.ts:25]: With this

**Review by tgymnich — APPROVED** [review]

**Comment by tgymnich** [comment]
Thank you

**Pull request merged**